test_schemas/en/test_custom_page_titles.json

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
You should add a section summary to demo it working on the summary screen.

Reply:
Added a repeating section summary test to also use the list item index in 7e426eac6 and 3e64c291